libsrc/_DEVELOPMENT/target/clib_variables.inc

```diff
@@ -38,27 +38,55 @@ IF __clib_malloc_heap_size = -1
       
    ld hl,-1                    ; ensure sp == 0 is accepted
    add hl,sp                   ; hl = top of the stack
-   
+
+IF __CPU_INTEL__ || __CPU_GBZ80__
+IF __CPU_8085__
+   ld bc,__BSS_END_tail
+   sub hl,bc                   ; hl = space between top of stack and end of bss - 1
+ELSE
+   ld de,__BSS_END_tail
+   ld a,l
+   sub e
+   ld a,h
+   sbc d                       ; hl = space between top of stack and end of bss - 1
+ENDIF
+ELSE
    ld de,__BSS_END_tail
-   
    xor a
    sbc hl,de                   ; hl = space between top of stack and end of bss - 1
+ENDIF
+
+   jp C, __Exit                ; if stack overlaps bss, this is a fatal error
 
-   jp c, __Exit                ; if stack overlaps bss, this is a fatal error
-   
    ld bc,__crt_stack_size + 14 ; amount of space reserved by stack + minimum heap size - 1
+
+IF __CPU_INTEL__ || __CPU_GBZ80__
```

Comment:
To avoid the nesting maybe:

IF __CPU_8085__

ELIF __CPU_GBZ80__ | __CPU_8080__

ELSE
  ;; z80 path
ENDIf

Reply:
Thanks.. didn't know ELIF was accepted. I didn't notice it being used.
There's quite a bit of nesting needed otherwise.